server/plugin/command.go

```diff
@@ -85,6 +88,99 @@ func (p *Plugin) getGithubClient(userInfo *GitHubUserInfo) *github.Client {
 	return p.githubConnect(*userInfo.Token)
 }
 
+func (p *Plugin) getMutedUsernames(userInfo *GitHubUserInfo) []string {
+	mutedUsernameBytes, err := p.API.KVGet(userInfo.UserID + "-muted-users")
+	if err != nil {
+		return nil
+	}
+	mutedUsernames := string(mutedUsernameBytes)
+	var mutedUsers []string
+	if len(mutedUsernames) == 0 {
+		return mutedUsers
+	}
+	mutedUsers = strings.Split(mutedUsernames, ",")
+	return mutedUsers
+}
+
+func (p *Plugin) handleMuteList(args *model.CommandArgs, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	return "Your muted usernames: " + strings.Join(mutedUsernames, ", ")
+}
+
+func (p *Plugin) handleMuteAdd(args *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
```

Comment:
It would me nice to add a check here if the provided username is valid. But that could be a follow up HW.

Reply:
I didn't add anything crazy thorough, but realized that a `,` character in a username could break the parsing of the CSV values. I added a check for that now. Everything else should be fine.